langchain/document_loaders/parsers/html/bs4.py

```diff
@@ -0,0 +1,56 @@
+"""Loader that uses bs4 to load HTML files, enriching metadata with page title."""
+
+import logging
+from typing import Any, Dict, Iterator, Mapping, Optional, Union
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseBlobParser
+from langchain.document_loaders.blob_loaders import Blob
+
+logger = logging.getLogger(__name__)
+
+
+class BS4HTMLParser(BaseBlobParser):
+    """Parser that uses beautiful soup to parse HTML files."""
+
+    def __init__(
+        self,
+        features: str = "lxml",
+        bs_kwargs: Optional[Mapping[str, Any]] = None,
```

Comment:
ooc (out of curiosity) how do you think about passing in kwargs as dict vs **kwargs?

Reply:
why not :)

My main concern tends to be that there might be another function that'll require them... but here maybe it's OK to assume it goes into the initializer by default, and everything else has to be explicitly specified